eng/pipelines/templates/stages/archetype-rust-release.yml

```diff
@@ -145,6 +149,8 @@ stages:
               CommitMsg: "Increment package version after release of ${{ artifact.name }}"
               PRTitle: "Increment versions for ${{parameters.ServiceDirectory}} releases"
               CloseAfterOpenForTesting: '${{parameters.TestPipeline}}'
+              ${{ if startsWith(variables['Build.SourceBranch'], 'refs/pull/') }}:
```

Comment:
I thought PRs happen in a `refs/merge` prefix?

Reply:
pr refs look like  `refs/pull/1234/(head|merge)`